.github/workflows/pull_request.yml

```
@@ -60,11 +58,8 @@ jobs:
 
       - name: Validate SPDX 3.0 Documents
         run: |
-          SPDX30_SCHEMA_URL="https://spdx.org/schema/3.0.0/spdx-json-schema.json"
-          SPDX30_SHACL_URL="https://spdx.org/rdf/3.0.0/spdx-model.ttl"
-
           for f in $(find . -type f -path '*/spdx-3.0/*.json'); do
```

```suggestion
          for f in $(find . -type f -path '*/spdx3.0/*.json'); do
```

Update the dir name. This may help resolve the conflict.